apps/crn-server/src/controllers/research-outputs.ts

```diff
@@ -79,46 +79,51 @@ export default class ResearchOutputs implements ResearchOutputController {
   async create(
     researchOutputCreateData: ResearchOutputCreateData,
   ): Promise<ResearchOutputResponse | null> {
-    await this.validateResearchOutput(researchOutputCreateData);
+    const normalizedResearchOutputCreateData = this.normaliseResearchOutputData(
+      researchOutputCreateData,
+    ) as ResearchOutputCreateData;
```

Comment:
I'm not a big fan of casting, try not typing the return type of the `normaliseResearchOutputData` method - maybe typescript can infer it in such a way that you then don't need to cast. Otherwise you should probably use some conditional magic or overloads to make sure that with `ResearchOutputCreateData` you get back `ResearchOutputCreateData` again.

Reply:
I'd make it a generic, so `this.normaliseResearchOutputData<ResearchOutputCreateData>(researchOutputCreateData)`